src/Controller/api/system.c

```diff
@@ -177,6 +177,7 @@ const char *wbu_system_webots_tmp_path() {
       struct dirent *entry;
       char folder_start[32];
       sprintf(folder_start, "webots-%d", webots_pid);
+      webots_pid = 0;
```

Comment:
it would probably good to add comment here, for example (not sure if this is really correct):
```suggestion
      // reset webots_pid containing the webots script process id to before retrieving the webots binary id
      webots_pid = 0;
```